src/test/osd/osd-scrub-repair.sh

```diff
@@ -314,27 +314,35 @@ function TEST_list_missing_erasure_coded() {
 
     # Put an object and remove the two shards (including primary)
     add_something $dir $poolname OBJ0 || return 1
-    local -a osds=($(get_osds $poolname OBJ0))
-
-    pids=""
-    run_in_background pids objectstore_tool $dir ${osds[0]} OBJ0 remove
-    run_in_background pids objectstore_tool $dir ${osds[1]} OBJ0 remove
-    wait_background pids
-    return_code=$?
-    if [ $return_code -ne 0 ]; then return $return_code; fi
-
+    local -a osds0=($(get_osds $poolname OBJ0))
 
     # Put another object and remove two shards (excluding primary)
     add_something $dir $poolname OBJ1 || return 1
-    local -a osds=($(get_osds $poolname OBJ1))
+    local -a osds1=($(get_osds $poolname OBJ1))
 
-    pids=""
-    run_in_background pids objectstore_tool $dir ${osds[1]} OBJ1 remove
-    run_in_background pids objectstore_tool $dir ${osds[2]} OBJ1 remove
-    wait_background pids
-    return_code=$?
-    if [ $return_code -ne 0 ]; then return $return_code; fi
+    # Stop all osd daemons
+    for id in $(seq 0 2) ; do
+        kill_daemons $dir TERM osd.$id >&2 < /dev/null || return 1
+    done
 
+    id=${osds0[0]}
+    ceph-objectstore-tool --data-path $dir/$id --journal-path $dir/$id/journal \
+        OBJ0 remove || return 1
+    id=${osds0[1]}
+    ceph-objectstore-tool --data-path $dir/$id --journal-path $dir/$id/journal \
```

Comment:
@dzafman maybe we can switch to https://github.com/ceph/ceph/pull/9203/commits/b6d30ec9e9580bb185b7301bd9bfce1fbd29d6b6#diff-be29cc5661964533b5db541336446042R78 for faster testing?